packages/testcontainers/src/containers/RegTestContainer/LoanContainer.ts

```diff
@@ -1,7 +1,19 @@
+import { MasterNodeKey, RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { StartOptions } from '../DeFiDContainer'
 import { MasterNodeRegTestContainer } from './Masternode'
 
 /**
  * @deprecated use MasterNodeRegTestContainer instead
  */
 export class LoanMasterNodeRegTestContainer extends MasterNodeRegTestContainer {
+  constructor (masternodeKey: MasterNodeKey = RegTestFoundationKeys[0]) {
+    super(masternodeKey, 'defi/defichain:HEAD-b810eda')
```

Comment:
This PR is merged to main few hours ago. However, when I connect to `MasterNodeRegTestContainer` directly, funny error returned.